File: app/src/main/java/com/panwar2001/pdf2txt/PreviewPDF.java
package com.panwar2001.pdf2txt;

import androidx.annotation.Nullable;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;

import com.google.android.gms.ads.AdRequest;
import com.panwar2001.pdf2txt.databinding.ActivityPreviewPdfBinding;


public class PreviewPDF extends AppCompatActivity {
    ActivityPreviewPdfBinding binding;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityPreviewPdfBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setSupportActionBar(binding.toolbar.toolbar);
        ActionBar actionBar=getSupportActionBar();
        if(actionBar!=null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
            Intent intent = getIntent();
            binding.pdfView.fromUri(intent.getData()).spacing(5).load();
        }
        //adview
        AdRequest adRequest = new AdRequest.Builder().build();
        binding.adView.loadAd(adRequest);
    }
    @Override
    public boolean onOptionsItemSelected(MenuItem item){
        int id=item.getItemId();
        if(id==android.R.id.home){
            finish();
            return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/com/panwar2001/pdf2txt/PreviewPDF.kt
package com.panwar2001.pdf2txt

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import com.google.android.gms.ads.AdRequest
import com.panwar2001.pdf2txt.databinding.ActivityPreviewPdfBinding

class PreviewPDF : AppCompatActivity() {
    private var binding: ActivityPreviewPdfBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityPreviewPdfBinding.inflate(
            layoutInflater
        )
        setContentView(binding!!.getRoot())
        setSupportActionBar(binding!!.toolbar.toolbar)
        val actionBar = supportActionBar
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true)
            val intent = intent
            binding!!.pdfView.fromUri(intent.data).spacing(5).load()
        }
        //adview
        val adRequest = AdRequest.Builder().build()
        binding!!.adView.loadAd(adRequest)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
            return true
        }
        return super.onOptionsItemSelected(item)
    }
}